### PR TITLE
Add callbacks to EMFReader for GUI

### PR DESCRIPTION
**Pull request by DouglasHalse**

### CODE REVIEW

File: SpookStationGui/main.py [hunk 1]
```diff
@@ -25,11 +27,24 @@ def __init__(self, deviceName, **kwargs):
...
+        self.ids['led' + str(led)].canvas.opacity = opacity
+        self.ids['led' + str(led)].canvas.ask_update()
 
     def SetLedState(self, ledState):
         for led in range(1, 5):
-            self.ids['led' + str(led)].canvas.opacity = 1 if  led <= ledState else  .5
+            opacity = 1 if  led <= ledState else  .5
+            Clock.schedule_once(partial(self.SetCanvasLedCanvasOpacity, led, opacity ), -1)
+
+    def setUseSoundActive(self, useSound, *largs):
```
Comment (Raisaur) [inline SpookStationGui/main.py:40]: _**S**_etUseSoundActive

File: SpookStationGui/main.py [hunk 2]
```diff
@@ -39,11 +54,17 @@ def OnFluctuationRateChanged(self, rate):
         deviceManager.devices[deviceManager.getDeviceIndex(self.deviceName)].setFluctuationRate(rate)
         print("Rate set to " + str(rate))
 
+    def onMuteButtonChange(self):
```
Comment (Raisaur) [inline SpookStationGui/main.py:57]: **O**nMuteButtonChange

**Review by Raisaur — CHANGES_REQUESTED** [review]
See comments

**Review by Raisaur — APPROVED** [review]
yes

**Comment by DouglasHalse** [comment]
Oh lord, I wanted to squash those commits before finishing this

**Pull request merged**